src-electron/util/types.js

```diff
@@ -78,6 +78,77 @@ async function typeSizeAttribute(db, zclPackageId, at, defaultValue = null) {
   }
 }
 
+/**
+ *
+ * @param {*} value
+ * @param {*} size
+ * @returns The big endian value for a given float value padded with
+ * the given size.
```

Comment:
This should probably document the "0x" prefix, that the value will be hex, etc, etc.